src/endpoints/files.js

```diff
@@ -8,26 +9,38 @@ class Files extends BaseExtend {
     this.endpoint = 'files'
   }
 
-  Create(body) {
-    let requestBody = body
+  Create(body, contentType = null) {
+    const additionalHeaders = {}
 
-    if (!(body instanceof FormData) && body.file_location) {
-      const formData = new FormData()
-      formData.append('file_location', body.file_location)
-
-      requestBody = formData
+    // Allows users to specify content type, useful if using Node and an
+    // implementation of FormData
+    if (contentType) {
+      additionalHeaders['Content-Type'] = contentType
     }
 
     return this.request.send(
       `${this.endpoint}`,
       'POST',
-      requestBody,
+      body,
+      undefined,
       undefined,
+      false,
       undefined,
-      false
+      additionalHeaders
     )
   }
 
+  Link(href) {
```

Comment:
`href` is Hypertext REFerence, so I would probably rename this to url or something instead.

Reply:
agree